src/me/components/DocSearchWidget.tsx

```diff
@@ -42,7 +42,7 @@ const DocSearchWidget: FC = () => {
       <DocSearch type={DocSearchType.Widget} />
       <p className="WidgetHelperText">Press CTRL + M on any page to search</p>
       <div className="useful-links">
-        <h4>Useful Links</h4>
+        <h4>USEFUL LINKS</h4>
```

Comment:
I think we should prefer `text-transform` over all caps: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

Here's a good article explaining why: https://medium.com/@mandy.michael/why-you-shouldnt-write-your-content-in-uppercase-instead-use-css-b03ac2c65b99

Reply:
✅